share/github-backup-utils/ghe-gc-disable

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
+#/ Usage: ghe-gc-disable [<option>...] <host>
+#/ Helper to disable and drain GC operations on a GitHub Enterprise server.
+set -e
+
+# Bring in the backup configuration
+cd $(dirname "$0")/../..
```

Comment:
The `cd` is already causing trouble in other scripts when installing `backup-utils` from the Debian package. Now that we're using `Bash`, can use use something like this instead to avoid altering `PWD`?

```
BASE_PATH="$( cd "$( dirname "${BASH_SOURCE[0]}" )/../.." && pwd )"
. share/github-backup-utils/ghe-backup-config
```